### PR TITLE
Update README.md for Feature 3

### PR DESCRIPTION
**Pull request by andrewjamesford**
This is my pull request for Feature 3

### CODE REVIEW

File: README.md
Comment (andrewjamesford) [inline README.md:1]: This is a comment for a file

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@
 This is my change to main
 
 This is my change for Feature 2
+
+This is my change for Feature 3
```
Comment (andrewjamesford) [inline README.md:8]: This the most **amazing** code ever @andrewjamesford 
```javascript
let yourVar = 6;
```

**Comment by andrewjamesford** [comment]
This should show straight away

**Review by andrewjamesford — COMMENTED** [review]
Thanks for that looks good to me

**Review by jericrealubit — APPROVED** [review]

**Pull request merged**